common/src/test/java/com/amazonaws/services/schemaregistry/common/DataPlatformSchemaNamingStrategyTest.java

```diff
@@ -0,0 +1,38 @@
+package com.amazonaws.services.schemaregistry.common;
+
+import org.apache.avro.Schema;
+import org.apache.avro.generic.GenericData;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+public class DataPlatformSchemaNamingStrategyTest {
+    @Test
+    public void test_dataPlatformSchemaNamingStrategy() throws IOException {
+        String fileName = "src/test/java/resources/avro/user3.avsc";
```

Comment:
The file `src/test/java/resources/avro/user3.avsc` is missing

Reply:
no